webroot/index.html

```diff
@@ -63,7 +63,6 @@
     <script type="preload" src="/js/web_modules/common/core-fed3ccd8.js"></script>
     <script type="preload" src="/js/web_modules/htm.js"></script>
     <script type="preload" src="/js/web_modules/@joeattardi/emoji-button.js"></script>
-    <script type="preload" src="/js/web_modules/@justinribeiro/lite-youtube.js"></script>
```

Comment:
Can you also remove the `lite-youtube` dependency from the [`web_modules` folder](https://github.com/owncast/owncast/tree/develop/webroot/js/web_modules)?

Reply:
If you only remove files from that directory they will automatically come back. Feel free to remove the dependency from build/javascript/package.json so it won't show back up.